### DISCUSSION TITLE
How to make junit reporter not hiding test failure log

### DISCUSSION THREAD

**Discussion by ahnpnl** [vitest-dev/vitest#2630] [Q&A]
Hi,

I'm using `junit `reporter and I noticed that whenever there are failed tests, the failure log is not printed out. Is there a way to fix this issue?

**Comment by sheremet-va** [comment]
If you create a small issue with reproduction, I can look into it 🤔

**Reply by ahnpnl** [reply]
Hi, sorry for a delay, here is the reproduce case https://stackblitz.com/edit/vitest-dev-vitest-uqd5qv?file=vite.config.ts 

I think it's related to the fact that I use yarn workspace.
